github-notifications/requirements.txt

```diff
@@ -0,0 +1,4 @@
+functions-framework==3.*
+flask==2.1.0
+slackclient==2.9.4
+python-dotenv==0.21.0
```

Comment:
I ran into some issues with missing the requests library. This could totally be user-error from me being not as well-versed in the python world, but I feel like maybe we need it listed here?